core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -596,7 +597,11 @@ public FileStatus[] listStatus(Path path) throws IOException {
     AlluxioURI uri = getAlluxioPath(path);
     List<URIStatus> statuses;
     try {
-      statuses = mFileSystem.listStatus(uri);
+      ListStatusPOptions listStatusPOptions = ListStatusPOptions.getDefaultInstance().toBuilder()
```

Comment:
let's fetch this property in the cluster instead of fetching it every time when the method is called.

Reply:
How can the cluster get the attributes set by a user?  It may be different for each user. It seems can't do it now.